charts/cbcontainers-agent/cbcontainers-agent-chart/example-values.yaml

```diff
@@ -0,0 +1,188 @@
+# accessToken is the API token used by the agent to communicate with the backend
+accessToken: ""
+# accountId is the ID of the organization account
+accountId: ""
```

Comment:
orgKey?

Reply:
do you want me to rename it?

Reply:
I'm suggesting this yes, because the customer is not aware on how we use 'account'
it something from the Octarine days

Reply:
done